### PR TITLE
chore: qr-code benchmark

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: benchmarks/qr-code/index.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      }
+    }
+  ]
+};
+
+/**
+ * wrap a document and automatically the fields with random data. data can contain any characters provided as an argument.
+ * Two pre-defined sets of characters exist, one with latin characters, one with chinese characters.
+ * the fields name is always using the latin characters set.
+ * every field name is 8 characters length, and every value is 20 characters length. It could be made configurable.
```
Comment (cavacado) [inline benchmarks/qr-code/index.ts:65]: for the field name, thats not configurable right?
the wording makes it seem like its configurable

File: benchmarks/qr-code/readme.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Key decisions
+
+We took some decisions to create this benchmark:
+
+- OpenAttestation credentials expect to hold UTF-8 data. Thus, we can only use the [bytes mode](https://www.npmjs.com/package/qrcode#qr-code-capacity) to store data inside the Qr code
+- We noticed that using the [L error correction level](https://www.npmjs.com/package/qrcode#error-correction-level) could produce an unreadable QR Code, especially when reaching the capacity limit. We decided to stick with the M error correction level.
+- All OpenAttestation credentials expect a minimum set of data to work. In this benchmark, we added information about `one` issuer and the template. While the template is not a mandatory field, it enables a key feature of OpenAttestation credential, decentralised rendering.
+- We add additional fields at the root of the document and not inside nested object:
+  - The keys use Latin character sets and are 9 characters in length.
```
Comment (cavacado) [inline benchmarks/qr-code/readme.md:23]: should be 8?

File: benchmarks/qr-code/index.ts (hunk 1 above)
Comment (Nebulis) [reply benchmarks/qr-code/index.ts:65]: It's not indeed, which part make you think it is?

File: benchmarks/qr-code/readme.md (hunk 2 above)
Comment (Nebulis) [reply benchmarks/qr-code/readme.md:23]: 8 random char + the index position

File: benchmarks/qr-code/readme.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# Goal
+
+The goal of the benchmark is just to give an overview of how many additional fields/data can be added into am OpenAttestation credential, before reaching the capacity limit of a QR code.
+
+In this document, we will refer to additional fields, as fields that are not initially required in an OpenAttestation credential.
+
+it is important to note that the result is just an overview and not the general truth. For instance, if the document states that it works for 13 additional fields, it's meant under specific conditions. You must **NOT** expect that it will always work with 13 additional fields. The input (size of the data, character set used) greatly impacts the result.
+
+# Key decisions
```
Comment (Nebulis) [inline benchmarks/qr-code/readme.md:15]: add capacity size

File: benchmarks/qr-code/readme.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Purpose
```
Comment (Nebulis) [inline benchmarks/qr-code/readme.md:1]: point out that images wont work

**Review by jley81 — APPROVED** [review]

**Pull request merged**